### PR TITLE
[IMP] resource_stock_information: reset values when back in stock

### PR DESCRIPTION
**Pull request by robinkeunen**
[task](https://gestion.coopiteasy.be/web?#id=7509&view_type=form&model=project.task&action=475&active_id=198)

- reset values when putting a value back to available
- track resource information value chage

### CODE REVIEW

File: resource_planning/models/resource_resource.py [hunk 1]
```diff
@@ -68,18 +69,15 @@ def _check_user_type_fields(self):
 
     @api.multi
     def action_unavailable(self):
-        for resource in self:
-            resource.state = "unavailable"
+        self.write({"state": "unavailable"})
```
Comment (victor-champonnois) [inline resource_planning/models/resource_resource.py:72]: What is the reason for using `write` instead of replacing the attributes ? I don't understand the implications.

Also is this related to the task or is it unrelated refactoring (in which case 
shouldn't it be put in a separate commit ?) ?

**Comment by victor-champonnois** [comment]
I read the code, and everything seemed clear and OK to me :)

File: resource_planning/models/resource_resource.py (hunk 1 above)
Comment (robinkeunen) [reply resource_planning/models/resource_resource.py:72]: It's unrelated _en passant_ refactoring indeed. Avoiding the loops sends one query to the db for all records

**Comment by robinkeunen** [comment]
@coccinouille gimme the green thumb if you agree w/ my last changes 🙏
